src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/EnvelopeMessager.kt

```diff
@@ -30,6 +31,10 @@ class EnvelopeMessager( @Autowired client: QueueClient) : IQueueClient by client
             updateCaseData.put("po_box", poBox)
         }
 
+        val documents = updateCaseData.get("documents") as JSONArray
+        val document = documents.get(0) as JSONObject
```

Comment:
what if there's more than one document?

Reply:
Good point. I Updated the helper class to return the list of urls.

Reply:
@rkondratowicz atm we have only one document in the functional tests. I'll change this method to support multiple documents as part of the next functional test